taskcluster/kinds/module-build/kind.yml

```diff
@@ -33,6 +33,7 @@ task-defaults:
       - [source, taskcluster/scripts/toolchain/cross-compile-setup.sh]
       - [source, taskcluster/scripts/toolchain/copy-libs-dir.sh, libs]
       - [bash, '-c', 'echo "rust.targets=arm,arm64,x86_64,x86,darwin,linux-x86-64\n" > local.properties']
+      - [bash, '-c', 'find']
```

Comment:
Did you want to include this in landed version still? Adds about 4500 lines to the log file

Reply:
Nope, thanks for the catch.